README.md

```diff
@@ -1,39 +1,43 @@
+
 # Con-PCA Client
-### Angular frontend application
 
-### Requirements
+## Requirements
```

Comment:
We're pretty strict about including closing octothorpes for Markdown headers, e.g. `### Requirements ###`.  Can you add those where needed in this file?

Reply:
@jsf9k I have added those closing octothorpes to all headers.